docs/projects/amt/reporting-standard/schemas/assessmentcard_schema.json

```diff
@@ -0,0 +1,86 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://raw.githubusercontent.com/MinBZK/ai-validatie/main/docs/projects/amt/reporting-standard/schemas",
+    "title": "Model card schema",
+    "description": "JSON schema for model card",
+    "type": "object",
+    "properties": {
+        "provenance": {
+            "type": "object",
+            "properties":{
+                "git_commit_hash":{
+                    "type": "string",
+                    "description": "Git commit hash of the commit which contains the transformation file used to create this card"
+                },
+                "timestamp":{
+                    "type": "string",
+                    "description": "A timestamp of the date, time and timezone of generation of this Assessment Card in ISO 8601 format"
+                },
+                "uri": {
+                    "type": "string",
+                    "description": "URI to the tool that was used to perform the transformations"
+                },
+                "author":{
+                    "type": "string",
+                    "description": "Name of person that initiated the transformations"
+                }
+            }
+        },
+        "name":{
+            "type": "string",
+            "description": "Name of the assessment"
+        },
+        "urn":{
+            "type": "string",
+            "description": "A Uniform Resource Name (URN) of the instrument in the instrument register"
+        },
+        "date":{
+            "type": "string",
```

Comment:
Maybe add the format for a datetime here too

Reply:
We can even enforce the format: https://json-schema.org/understanding-json-schema/reference/string#dates-and-times